lib/Doctrine/ODM/MongoDB/Mapping/Annotations/File/ChunkSize.php

```diff
@@ -4,19 +4,19 @@
 
 namespace Doctrine\ODM\MongoDB\Mapping\Annotations\File;
 
+use Attribute;
+use Doctrine\Common\Annotations\Annotation\NamedArgumentConstructor;
 use Doctrine\ODM\MongoDB\Mapping\Annotations\AbstractField;
 
 /**
  * @Annotation
+ * @NamedArgumentConstructor
  */
+#[Attribute(Attribute::TARGET_PROPERTY)]
 final class ChunkSize extends AbstractField
 {
-    /** @var string */
-    public $name = 'chunkSize';
-
-    /** @var string */
-    public $type = 'int';
-
-    /** @var bool */
-    public $notSaved = true;
+    public function __construct(?string $name = 'chunkSize')
+    {
+        parent::__construct($name, 'int', false, [], null, true);
+    }
```

Comment:
```suggestion
    public function __construct()
    {
        parent::__construct('chunkSize', 'int', false, [], null, true);
    }
```
Never used GridFS, but based on https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/reference/annotations-reference.html#file-chunksize and https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst looks like name is not configurable.

It that's right, the same applies to:

- [Filename](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/reference/annotations-reference.html#file-filename)
- [Length](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/reference/annotations-reference.html#file-length)
- [Metadata](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/reference/annotations-reference.html#file-metadata)
- [UploadDate](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/2.2/reference/annotations-reference.html#file-uploaddate)